### PR TITLE
non-English simplified mobile header and footer

### PR DESCRIPTION
**Pull request by Erin007**
Users viewing the mobile site in languages other than English should see a simplified version of the header and footer because not all of the options provided in the English header and footer are translated. 

Simple non-English mobile header
non-eng-mobile-header

Simple non-English mobile footer
non-eng-mobile-footer

Note: The original version of this work in PR #14158 caused a JS error on non-English mobile pages because eventListeners couldn't be applied to non-existent elements. PR #14277 reverted the original work.  This PR resolves the errors and restores the desired changes.

### CODE REVIEW

**Comment by Erin007** [comment]
cc @davidsbailey & @joshlory

File: pegasus/sites.v3/code.org/public/js/mobile-footer.js [hunk 1]
```diff
@@ -6,10 +6,13 @@ $(function () {
     $('#down').toggle();
   });
 
-  document.querySelector( '#footer-toggle' )
-    .addEventListener( "click", function () {
-    this.classList.toggle( 'active' );
-  });
+  var footerToggle = document.querySelector( '#footer-toggle' );
```
Comment (breville) [inline pegasus/sites.v3/code.org/public/js/mobile-footer.js:9]: pardon my ignorance, but what exactly are we toggling in the footer?

Comment (tanyaparker) [reply pegasus/sites.v3/code.org/public/js/mobile-footer.js:9]: Open/closing the footer menu?

Comment (Erin007) [reply pegasus/sites.v3/code.org/public/js/mobile-footer.js:9]: Yes, when there are all of the links, they are in a sort of drawer that opens and closes.

**Pull request merged**